src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -243,26 +243,19 @@
                     <img src="../assets/svg/icons-reject-small.svg" alt="Cancel" class="pr-2 pb-1" />Cancel
                   </button>
                   <button
-                    *ngIf="!workflow?.topic && !topicEditing"
                     [disabled]="
-                      forumUrlEditing || workflowPathEditing || defaultTestFilePathEditing || (isRefreshing$ | async) || !canWrite
+                      forumUrlEditing ||
+                      workflowPathEditing ||
+                      defaultTestFilePathEditing ||
+                      (isRefreshing$ | async) ||
+                      !canWrite ||
+                      workflow?.mode === WorkflowType.ModeEnum.DOCKSTOREYML
                     "
                     type="button"
                     mat-raised-button
                     class="small-mat-btn-skin accent-1-dark small-btn-structure mat-elevation-z"
                     (click)="toggleEditTopic()"
-                    data-cy="topicEditButton"
-                  >
-                    <img src="../assets/svg/icons-actions-edit.svg" class="pr-2" alt="Edit Topic Manual" />Edit
-                  </button>
-                  <button
-                    *ngIf="workflow?.topic || topicEditing"
-                    [disabled]="!canWrite || workflow?.mode === WorkflowType.ModeEnum.DOCKSTOREYML"
-                    type="button"
-                    mat-raised-button
-                    class="small-mat-btn-skin accent-1-dark small-btn-structure mat-elevation-z"
-                    (click)="toggleEditTopic()"
-                    data-cy="topicSaveButton"
+                    [attr.data-cy]="topicEditing ? 'topicSaveButton' : 'topicEditButton'"
                     [matTooltip]="
                       workflow?.mode === WorkflowType.ModeEnum.DOCKSTOREYML
                         ? 'Edit the manual topic for this ' +
```

Comment:
I only noticed this because of your PR description, and it's arguably beyond the scope of the PR, but I will suggest anyway to optionally reverse the order of the two sentences. Instead of:

"Edit the manual topic for this entry in the .dockstore.yml file on GitHub. Editing the topic in the UI is deprecated for GitHub App entries."

I would have, stating the reason for the disabling first, then the remedy:

"Editing the topic in the UI is deprecated for GitHub App entries. Edit the manual topic for this entry in the .dockstore.yml file on GitHub."